static/src/javascripts/boot.js

```diff
@@ -1,5 +1,12 @@
 // @flow
 
+// es7 polyfills not provided by pollyfill.io
+import 'core-js/modules/es7.object.values';
+import 'core-js/modules/es7.object.entries';
+import 'core-js/modules/es7.object.get-own-property-descriptors';
+import 'core-js/modules/es7.string.pad-start';
+import 'core-js/modules/es7.string.pad-end';
```

Comment:
string pad-end and pad-start will be in the next release 🎉  - https://github.com/Financial-Times/polyfill-service/pull/1117